Robot Code/src/org/usfirst/frc/team3316/robot/subsystems/Chassis.java

```diff
@@ -95,9 +98,11 @@ public void setMotors(double left, double right)
 
 	public boolean openLongPistons()
 	{
-		if (shortPistonsRight.get().equals(Value.kForward) || shortPistonsLeft.get().equals(Value.kForward))
+		if (shortPistonsRight.get().equals(Value.kForward)
+				|| shortPistonsLeft.get().equals(Value.kForward))
```

Comment:
Replace this with functions on Chassis such as isShortRightPistonOpen() or a more general isPistonOpen(PistonSide, PistonLength)
The current way doesn't allow us to decouple the forward/backward/off low-level details from the higher level "is piston open" status